android-components/samples/browser/src/main/java/org/mozilla/samples/browser/addons/AddonsFragment.kt

```diff
@@ -138,6 +138,14 @@ class AddonsFragment : Fragment(), AddonsManagerAdapterDelegate {
         requireContext().startActivity(intent)
     }
 
+    override fun shouldShowFindMoreAddonsButton() {
+        return false
+    }
+
+    override fun onFindMoreAddonsButtonClicked() {
+        // do nothing
+    }
```

Comment:
Nit: as we already provided a default value (`Unit`) in `AddonsManagerAdapterDelegate`, I think we may not need to provide an implementation here :)